metadata-ingestion/src/datahub/ingestion/source/looker/looker_common.py

```diff
@@ -109,10 +128,35 @@ def remove_quotes(cls, v):
         v = v.replace('"', "").replace("`", "")
         return v
 
+    def preprocess_file_path(self, file_path: str) -> str:
+        new_file_path: str = str(file_path)
+
+        str_to_remove: List[str] = [
+            ".view.lkml",
+        ]
+
+        str_to_replace: Dict[str, str] = {
+            f"imported_projects/{self.project_name}/": "",
+            "/": "_",  # / is not urn friendly
+        }
+
+        for remove in str_to_remove:
+            new_file_path = new_file_path.rstrip(remove)
```

Comment:
can't use rstrip here. It works on individual characters and not complete string.

Reply:
done